manifests/m/mar10/stressor/0.5.1/mar10.stressor.yaml

```diff
@@ -0,0 +1,18 @@
+#
+# Windows Package Manager Manifest
+#
+PackageIdentifier:  mar10.stressor
+PackageVersion:     0.5.1
+PackageLocale:      en-US
+Publisher:          'Martin Wendt'
+PackageName:        'stressor'
+License:            'MIT'
+LicenseUrl:         'https://raw.githubusercontent.com/mar10/stressor/master/LICENSE.txt'
+ShortDescription:   'Stress-test your web app'
+Installers:
+ - Architecture:    x64
+   InstallerType:   msi
+   InstallerUrl:    https://github.com/mar10/stressor/releases/download/v0.5.1/stressor-0.5.1.0-amd64.msi
+   InstallerSha256: 98B70F59583898761CC969C4389350153F8F14F93B8DF9A3C99CA2586DF98CD6
```

Comment:
```suggestion
   InstallerSha256: 98B70F59583898761CC969C4389350153F8F14F93B8DF9A3C99CA2586DF98CD6
   ProductCode: "{D4E5F9B7-CF19-455C-859D-7E121A7E4F1C}"
```

Reply:
Thanks,
is this supposed to by the 'upgrade code' that I use to build the MSI installer?
In this case I already have a GUID, that I would use as 'ProductCode':
```py
# See https://cx-freeze.readthedocs.io/en/latest/distutils.html#bdist-msi
bdist_msi_options = {
    "upgrade_code": "{3DA14E9B-1D2A-4D90-92D0-2375CF66AC3D}",
    "add_to_path": True,
}
```

Reply:
>is this supposed to be the 'upgrade code' that I use to build the MSI installer?

No, see https://docs.microsoft.com/en-us/windows/win32/msi/productcode.

Reply:
Got it, thank you.